mteb/evaluation/evaluators/Evaluator.py

```diff
@@ -1,11 +1,20 @@
 from abc import ABC, abstractmethod
+import random
 
+import numpy as np
+import torch
 
 class Evaluator(ABC):
     """
     Base class for all evaluators
     Extend this class and implement __call__ for custom evaluators.
     """
+    def __init__(self, **kwargs):
+        self.seed = kwargs.get("seed", 42)
+
+        random.seed(self.seed)
+        np.random.seed(self.seed)
+        torch.manual_seed(self.seed)
```

Comment:
Should we add `if use_cuda: torch.cuda.manual_seed_all(seed_value)` like in [this example?](https://discuss.pytorch.org/t/what-is-manual-seed/5939/16)